packages/devtools_app/lib/src/analytics/analytics.dart

```diff
@@ -475,12 +475,12 @@ void setupAndGaScreen(String screenName) async {
   }
 }
 
-void setupDimensions() {
+Future<void> setupDimensions() async {
```

Comment:
should we now be awaiting this function everywhere we are calling it?

Reply:
the only place we call this is in `DevToolsAppState.initState`, so we cannot await it there. Regardless of this though, I think it's better to have the proper return type here so we can await `computeFlutterClientId` before setting the `_analyticsComputed` bit to true at the end of this method.